examples/2.1.5.1.9-facet-json-range.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+require_once(__DIR__.'/init.php');
+htmlHeader();
+
+// create a client instance
+$client = new Solarium\Client($adapter, $eventDispatcher, $config);
+
+// get a select query instance
+$query = $client->createSelect();
+
+// get the facetset component
+$facetSet = $query->getFacetSet();
+
+// create a json range instance and set options
+// Set the 'other' parameter to retrieve before, after, between counts.
+// See https://lucene.apache.org/solr/guide/json-facet-api.html#range-facet-parameters
+$priceranges = new \Solarium\Component\Facet\JsonRange(['local_key' => 'priceranges', 'field' => 'price','start'=>1 ,'end'=>300,'gap'=>100,'other'=>'all']);
```

Comment:
In the example we should use the constant `JsonRange::OTHER_ALL` instead of 'all'.

Reply:
Have switched to constants and listed possible values in the example